### PR TITLE
Added "Silo" to Media > Image

### PR DESCRIPTION
**Pull request by josejuanqm**

### CODE REVIEW

**Comment by victorgama** [comment]
@vsouza @dkhamsing @lfarah Author states the lib is _under development_. Shall we add it right away or hold it back until a stable version is available?

**Comment by josejuanqm** [comment]
Hello!

Build is stable and working :)

**Comment by josejuanqm** [comment]
Added Ody

File: README.md [hunk 1]
```diff
@@ -329,6 +329,8 @@ Projects in Swift language will be marked with :large_orange_diamond: and :watch
  * [XCDYouTubeKit](https://github.com/0xced/XCDYouTubeKit) - YouTube video player for iOS, tvOS and OS X
  * [AVAnimator](http://www.modejong.com/AVAnimator/) - An open source iOS native library that makes it easy to implement non-trivial video/audio enabled apps.
  * [360 VR Player](https://github.com/hanton/HTY360Player) - A open source, ad-free, native and universal 360 degree panorama video player for iOS.
+ * [Silo] (https://github.com/josejuanqm/Silo) - Image loading framework with loaders. :large_orange_diamond:
```
Comment (victorgama) [inline README.md]: Do you mind removing those spaces between `]` and `(`? Thanks!

Comment (josejuanqm) [inline README.md]: Done! :+1:

Comment (victorgama) [inline README.md]: Perfect! Thank you.

**Comment by victorgama** [comment]
Thank you!

**Comment by josejuanqm** [comment]
Hey, umm i think i messed up, it was supposed to be on images and its on video :( is there a way to fix that or do i have to make a new pull request? im sorry

**Comment by victorgama** [comment]
Oh, don't worry. I already moved both projects to the right category. Thank you for the heads up.

**Comment by josejuanqm** [comment]
Thank you!

**Pull request merged**